Dockerfile

```diff
@@ -38,18 +37,13 @@ WORKDIR ${MC_HOME}
 ADD licenses /licenses
 
 ### Atomic Help File
-COPY description.md /tmp/
+COPY help.1 /help.1
 
 RUN dnf config-manager --disable && \
     dnf update -y  && rm -rf /var/cache/dnf && \
     dnf -y update-minimal --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs && \
 ### Add your package needs to this installation line
     dnf -y --setopt=tsflags=nodocs install java-11-openjdk wget unzip &> /dev/null && \
-### Install go-md2man to help markdown to man conversion
-    dnf -y --setopt=tsflags=nodocs install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm &> /dev/null && \
-    dnf -y --setopt=tsflags=nodocs install golang-github-cpuguy83-go-md2man &> /dev/null && \
-    go-md2man -in /tmp/description.md -out /help.1 && \
-    dnf -y remove golang-github-cpuguy83-go-md2man && \
```

Comment:
wasn't it necessary for some internal opneshift doc generation? cc: @hasancelik

Reply:
@hasancelik , you can confirm, but I think the only generated file was `/help.1` which I hardcoded.

I think we don't need to install the whole toolkit just to generate two lines of mandoc ;) But maybe I'm wrong and something else is generated?

Btw. this tool breaks multi-arch build.

Reply:
@leszko @mesutcelik,
> wasn't it necessary for some internal opneshift doc generation?
Afaik, it is not necessary for internal operations. `go-md2man` converts `description.md` into `help.1`. As Rafal said, `help.1` is hardcoded so we do not need to conversion step.